main.py

```diff
@@ -10,6 +10,13 @@
 BOT_TOKEN = getenv('BOT_TOKEN')
 CHANNEL_ID = getenv('CHANNEL_ID')
 
+# LOG_LEVEL can either be an int or a string
+LOG_LEVEL = getenv('LOG_LEVEL')
+if LOG_LEVEL.isnumeric:
```

Comment:
take care! `isnumeric()` is a method and then must be called!
```suggestion
if LOG_LEVEL.isnumeric():
```